### PR TITLE
Glue our translations to the simulator.

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**
It doesn't need much text so no need for it to have a translation
mechanism of its own. Could revisit if there's third-party demand - if we had translations at that point we could just inline them.

### CODE REVIEW

File: src/simulator/Simulator.tsx [hunk 1]
```diff
@@ -33,10 +33,10 @@ const Simulator = ({
   minWidth,
 }: SimulatorProps) => {
   // This needs the domain to be updated before we release.
-  const url = "https://stage-python-simulator.microbit.org/simulator.html";
+  //const url = "https://stage-python-simulator.microbit.org/simulator.html";
   // For testing with sim branches:
-  //const branch = "whatever";
-  //const url = `https://review-python-simulator.microbit.org/${branch}/simulator.html`;
+  const branch = "i18n-1";
+  const url = `https://review-python-simulator.microbit.org/${branch}/simulator.html`;
```
Comment (microbit-matt-hillsdon) [inline src/simulator/Simulator.tsx:39]: Needs reverting before merge once https://github.com/microbit-foundation/micropython-microbit-v2-simulator/pull/44 lands.

File: src/simulator/Simulator.tsx [hunk 2]
```diff
@@ -118,4 +121,21 @@ const Simulator = ({
...
+  const config = {
+    language: intl.locale,
+    translations: Object.fromEntries(
+      ["button-a", "button-b", "touch-logo", "start-simulator"].map((k) => [
+        k,
+        intl.formatMessage({ id: "simulator-" + k }),
+      ])
+    ),
+  };
+  console.log("Reconfiguring sim", config);
```
Comment (microbit-matt-hillsdon) [inline src/simulator/Simulator.tsx:137]:
```suggestion
```

Debug.

File: src/simulator/Simulator.tsx (hunk 1 above)
Comment (microbit-matt-hillsdon) [reply src/simulator/Simulator.tsx]: 4745559a

File: lang/ui.en.json [hunk 3]
```diff
@@ -931,6 +943,10 @@
     "defaultMessage": "Sound level",
     "description": "Sound level title"
   },
+  "simulator-start-simulator": {
+    "defaultMessage": "Button A",
```
Comment (microbit-robert) [inline lang/ui.en.json:947]:
```suggestion
    "defaultMessage": "Start simulator",
```

File: lang/ui.en.json [hunk 4]
```diff
@@ -779,6 +779,14 @@
     "defaultMessage": "Simulator actions",
     "description": "Aria label for the bar with simulator actions"
   },
+  "simulator-button-a": {
+    "defaultMessage": "Button A",
+    "description": "Button A aria label on the simulator board"
+  },
+  "simulator-button-b": {
+    "defaultMessage": "Button B",
+    "description": "Button A aria label on the simulator board"
```
Comment (microbit-robert) [inline lang/ui.en.json:788]:
```suggestion
    "description": "Button B aria label on the simulator board"
```

**Review by microbit-robert — APPROVED** [review]
Looks good now.

**Pull request merged**